### ISSUE TITLE
multiple definition of `nest::names::theta'

### ISSUE THREAD

**Issue by ikitayama** [jakobj/nest-simulator#79]
At the tip of Jakob's 5g branch, build of NEST on an x86 box gives:

```
[100%] Linking CXX executable nest
../topology/libtopology.a(topology_names.cpp.o):(.data+0x8): multiple definition of `nest::names::theta'
../nestkernel/libnestkernel.a(nest_names.cpp.o):(.data+0x124): first defined here
make[2]: *** [nest/nest] Error 1
make[1]: *** [nest/CMakeFiles/nest.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by ikitayama** [comment]
@jakobj could you cherry-pick commit https://github.com/ikitayama/nest-simulator/commit/11d5c5c07fdc16a263fcc4305dcee8511f162c00

**Comment by jakobj** [comment]
Fixed by #80 and closing
